Lib/test/test_tempfile.py

```diff
@@ -886,6 +886,19 @@ def make_file():
             self.assertEqual(l, lines[i])
         self.assertEqual(i, len(lines) - 1)
 
+    def test_next(self):
+        lines = [b'spam\n', b'eggs\n', b'beans\n']
+        with tempfile.NamedTemporaryFile(mode='w+b') as fd:
+            fd.write(b''.join(lines))
+            fd.seek(0)
+
+            for i, l in enumerate(lines):
+                self.assertEqual(l, next(fd))
+            self.assertEqual(i, len(lines) - 1)
```

Comment:
This can be collapsed to `self.assertEqual(list(fd), lines)` I'd say.

Reply:
fixed